qiita_db/handlers/tests/test_processing_job.py

```diff
@@ -192,7 +240,7 @@ def test_post_job_success(self):
              ]})
         obs = self.post(
             '/qiita_db/jobs/bcc7ebcd-39c1-43e4-af2d-822e3589f14d/complete/',
-            payload)
+            payload, headers={'Authorization': 'Bearer ' + self.token})
```

Comment:
To avoid more code duplication - in the base class you can add `self.headers = {'Authorization': 'Bearer ' + self.token}`, so we only need to do headers=headers in these calls. Either that or add a custom post that will help us to reduce all these calls...